rpcs3/Emu/Cell/Modules/sceNp.h

```diff
@@ -844,10 +845,20 @@ struct SceNpOnlineId
 struct SceNpId
 {
 	SceNpOnlineId handle;
-	u8 opt[8];
+
+	union
+	{
+		// This field (system reserved) seems to be combined of two parts
+		// The second is used by sceNpUtilSetPlatformType and sceNpUtilGetPlatformType
+		u8 opt[8];
+		se_t<u32, false, 1> unk1[2];
```

Comment:
For now I think it is better to use them as unaligned u32 (instead of two char[4]), it is much nicer to use this way.